reference/orders.v2.oas2.yml

```diff
@@ -99,6 +100,8 @@ paths:
 
         After the update, the PUT request clears all discounts and promotions applied to the changed order line items. Since the order data syncs with other ERP systems, like Amazon or eBay, the updated order returns to the default setting, removing any applied discounts.
         
+        To update the fees for an order, include the fee `id` in the request body. The body should include all relevant fields, with either the updated values or the existing values you wish to retain. Fees not included in the payload will be deleted. Fees provided in the request will be updated, and fees without an ID will be created as new.
```

Comment:
🍹 Probably we can include the consequence for fields not included in the request body.
```suggestion
        To update the fees for an order, include the fee `id` in the request body. The body should include all relevant fee fields, with either the updated values or the existing values you wish to retain (missing fee fields will be reset to empty value). Fees not included in the payload will be deleted. Fees provided with an ID in the request will be updated, and fees without an ID will be created as new.
```

Reply:
For missing fields, fee will actually throw 400
<img width="835" alt="image" src="https://github.com/user-attachments/assets/86ea5e11-9bf9-4462-b0ea-a8032985f0af">